### PR TITLE
Dismissing sidemenu gesture and Adding of LaTeX markup text blocks

### PR DESCRIPTION
**Pull request by sebastiantoh**
Changes:
- Allow dismissing side menu by swiping outside of SideMenu (while not interfering with annotations)
- Allow adding of LaTeX markup text block (done by introducing a new button)

### CODE REVIEW

File: Dynavity/Dynavity/view/ToolbarView.swift [hunk 1]
```diff
@@ -50,16 +50,27 @@ struct ToolbarView: View {
...
+            Button(action: {
+                viewModel.addMarkUpTextBlock(markupType: .latex)
+            }) {
+                Label("LaTeX", systemImage: "doc.richtext")
+            }
         }
-        label: {
-            Label("Add", systemImage: "plus")
-        }
+
```
Comment (ianyong) [inline Dynavity/Dynavity/view/ToolbarView.swift]:
```suggestion
```

Random newline.

**Review by ianyong — DISMISSED** [review]
LGTM! Should the user also be allowed to tap the grayed out area to close the side menu?

**Comment by sebastiantoh** [comment]
> LGTM! Should the user also be allowed to tap the grayed out area to close the side menu?

Yep! But I'll implement it some other time since I'm not sure if it'll interfere with any future tap gestures in the actual side menu

**Comment by ianyong** [comment]
> > LGTM! Should the user also be allowed to tap the grayed out area to close the side menu?
> 
> Yep! But I'll implement it some other time since I'm not sure if it'll interfere with any future tap gestures in the actual side menu

Ah ok, sounds good! Thanks!

**Review by ianyong — APPROVED** [review]

**Pull request merged**